src/packagedcode/golang.py

```diff
@@ -133,3 +138,28 @@ def build_gomod_package(gomod_data):
         homepage_url=homepage_url,
         dependencies=package_dependencies
     )
+
+
+def build_gosum_package(gosum_objs):
+    """
+    Return a Package object from a go.sum file.
+    """
+    package_dependencies = []
+    for obj in gosum_objs:
```

Comment:
Avoid using generic variable names like "obj" when possible... here we have a `Gosum` instance, so call it `gosum`

Reply:
:+1: